src/commands/Register.ts

```diff
@@ -118,15 +118,28 @@ export class Register extends CommandClient {
 
 export async function executeRegister(registerOptions?: RegisterOptions) {
   const { app, contracts, blockchain } = getServerConfig();
-  let registerConfig: RegisterConfig | undefined = undefined;
-  if (config.has('register')) {
-    registerConfig = config.get('register');
+
+  if (!config.has('register')) {
+    throw Error(`
+    Cannot find key "register" in the configuration.
+    Make sure registration is configured in "config"
+    folder for your network, or in the default configuration.
+    `);
   }
+  const { account, stake, funds, mnemonic }: RegisterConfig =
+    config.get('register');
```

Comment:
```suggestion
    config.get< RegisterConfig >('register');
```

Reply:
no need for this as the type is resolved from the type given to the const.